eng/Packages.Data.props

```diff
@@ -82,7 +82,7 @@
         - https://github.com/Azure/azure-functions-host/blob/dev/test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json
   -->
 
-  <ItemGroup Condition="'$(IsClientLibrary)' == 'true' or '$(IsGeneratorLibrary)' == 'true'">
+  <ItemGroup Condition="'$(IsClientLibrary)' == 'true' or '$(IsGeneratorLibrary)' == 'true' or '$(IsGeneratedSample)' == 'true' ">
```

Comment:
Do you want the generated samples to only use the track 2 approved packages?

This by the way will also allow other track 2 client libraries to consume these packages I'm not sure if really want that here.

Reply:
I can't make the generated sample projects test projects otherwise they don't use analyzers and pull in all kinds of stuff that isn't needed because these aren't test projects.